docs/user/security/authorization/index.asciidoc

```diff
@@ -139,17 +141,17 @@ Features are available to users when their roles grant access to the features, *
 
 ==== Assigning different privileges to different spaces
 
-Using the same role, it’s possible to assign different privileges to different spaces. After you’ve added privileges, click **Add {kib} privilege**. If you’ve already added privileges for either *** Global (all spaces)** or an individual space, you will not be able to select these in the **Spaces** selection control.
+Using the same role, it’s possible to assign different privileges to different spaces. After you’ve added privileges, click **Add {kib} privilege**. If you’ve already added privileges for either *** All Spaces** or an individual space, you will not be able to select these in the **Spaces** selection control.
```

Comment:
I know the UI uses *** All Spaces**, but can we simply use **All Spaces** here because it looks like an error with the leading *.